### ISSUE TITLE
[BUG] no such file error

### ISSUE THREAD

**Issue by dtnewman** [HazyResearch/meerkat#360]
I'm running the demo code and i get 

```
FileNotFoundError: [Errno 2] No such file or directory: '/Users/sabrieyuboglu/.meerkat/keys/.openai'
```

Looking at the code, i think the issue is here: https://github.com/HazyResearch/meerkat/blob/b55f8212ed825faaf93dfd9a8a6845af5f7afd70/meerkat/ops/complete.py#L43

**Comment by krandiash** [comment]
Do you mind pasting in what you ran, we can repro it

**Comment by dtnewman** [comment]
sure. I did it in an ipython notebook, but i'll copy paste the cells here


```
import meerkat as mk
import os
import json

# set env variable
os.environ['OPENAI_API_KEY']  

with open('extracted_data.json') as f:
    extracted_data = json.load(f)

data = [dict(case_number=x[0], outcome=x[1], conclusion=x[2]) for x in extracted_data]
df = mk.DataFrame(data)


mk.gui.start(api_port=2023)


df['file_location'] = df['case_number'].apply(lambda x: './documents/' + x.lower() + "__merits_decision.pdf")


df["pdf"] = mk.files(df['file_location'], "pdf")



df.gui.gallery(main_column="pdf")


# go through each row of the dataframe and check if the file exists and remove otherwise

for row in df.iterrows():
    file_location = row['file_location']
    print(file_location)
    if not os.path.isfile(file_location):
        # remove the row from the dataframe
        df = df[df['file_location'] != file_location]


flash = mk.gui.contrib.FlashFill(df=df, target_column="outcome2")


df['outcome3'] = mk.complete(df, prompt=flash.prompt, engine="openai/text-davinci-003", batch_size=12)

```

that last line is where i got error:

```
---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
Cell In[16], line 1
----> 1 df['outcome3'] = mk.complete(df, prompt=flash.prompt, engine="openai/text-davinci-003", batch_size=12)

File [~/.virtualenvs/data_analysis_venv/lib/python3.10/site-packages/meerkat/ops/complete.py:43](https://vscode-remote+wsl-002bubuntu.vscode-resource.vscode-cdn.net/home/daniel/code/data-analysis/chatgpt_pdf_project/~/.virtualenvs/data_analysis_venv/lib/python3.10/site-packages/meerkat/ops/complete.py:43), in complete(df, prompt, engine, batch_size, use_ray, num_blocks, blocks_per_window, pbar)
     38 from manifest import Manifest
     40 client_name, engine = engine.split("/")
     41 manifest = Manifest(
     42     client_name=client_name,
---> 43     client_connection=open("/Users/sabrieyuboglu/.meerkat/keys/.openai").read(),
     44     engine=engine,
     45     temperature=0,
     46     max_tokens=1,
     47     cache_name="sqlite",
     48     cache_connection="/Users/sabrieyuboglu/.manifest/cache.sqlite",
     49 )
     51 def _run_manifest(rows: mk.DataFrame):
     52     out = manifest.run([prompt.format(**row) for row in rows.iterrows()])

FileNotFoundError: [Errno 2] No such file or directory: '/Users/sabrieyuboglu/.meerkat/keys/.openai'
```

**Comment by ad12** [comment]
thanks for the issues! we have added the fix in #362
